cldf/languages.csv

```diff
@@ -1,80 +1,80 @@
 ID,Name,Glottocode,Glottolog_Name,ISO639P3code,Macroarea,Latitude,Longitude,Family,SubGroup,Source
-Akuntsu,Akuntsu,akun1241,Akuntsu,aqz,South America,-12.8322,-60.9716,Tupian,Tuparí,
-Amanaye,Amanaye,aman1266,Amanayé,ama,South America,-3.8,-48.66,Tupian,Tupi-Guarani,
-Amondawa,Amondawa,amun1246,Amundava,adw,South America,-9.04023,-62.3064,Tupian,Tupi-Guarani,
-Anambe,Anambe,anam1249,Anambé,aan,South America,-2.71119,-49.303,Tupian,Tupi-Guarani,
-AnambeEhrenreich,Anambe Ehrenreich,anam1251,Anambe of Ehrenreich,,South America,-4.5,-50.5,Tupian,Tupi-Guarani,
+Akuntsu,Akuntsu,akun1241,,,,-12.8322,-60.9716,,Tuparí,
+Amanaye,Amanaye,aman1266,,,,-3.8,-48.66,,Tupi-Guarani,
+Amondawa,Amondawa,amun1246,,,,-9.04023,-62.3064,,Tupi-Guarani,
+Anambe,Anambe,anam1249,,,,-2.71119,-49.303,,Tupi-Guarani,
+AnambeEhrenreich,Anambe Ehrenreich,anam1251,,,,-4.5,-50.5,,Tupi-Guarani,
```

Comment:
This regression is an effect of not downloading the language data from the google sheet. Did you get an error message with instructions how to create a google API key to use as key file?

Reply:
I did not get any instructions with the error message, just a prompt for a Google API key file. I was able to run the cldfbench with glottolog version 4.3 and concepticon version 2.4.0 afterwards. I also updated pylexibank as you asked before running the code. Should I push the results or is there something I should do first?

Reply:
I already updated the files in this PR. So no need to do anything now.